InternetHospitalWebUI/src/app/config.ts

```diff
@@ -0,0 +1 @@
+export const configUrl = 'https://localhost:44390';
```

Comment:
Maybe it's better to rename configUrl variable to HOST_URL or ROOT_URL? 
And name constants in uppercase with underscore between words, please.